batchflow/models/tf/encoder_decoder.py

```diff
@@ -220,7 +244,8 @@ def encoder(cls, inputs, name='encoder', **kwargs):
         list of tf.Tensors
         """
         base_class = kwargs.pop('base')
-        steps, downsample, block_args = cls.pop(['num_stages', 'downsample', 'blocks'], kwargs)
+        steps, order, downsample, block_args = cls.pop(['num_stages', 'order', 'downsample', 'blocks'], kwargs)
+        order = ''.join([item[0] for item in order])
 
```

Comment:
what about using this in config directly? order: 'ub'

Reply:
You can use it, as well as list with full names of operations